client/public/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Tripadvisor Attraction Page Clone</title>
+  </head>
+  <body>
+    <div>Hi</div>
+    <div id="app">Test</div>
+    <div id="image-main"></div>
+    <div id="experiences"></div>
+    <div id="best-nearby"></div>
+    <div id="reviews"></div>
+    <script src="imageMain/bundle.js"></script>
+    <script src="exp/bundle.js"></script>
+    <script src="reviewsModule/bundle.js"></script>
+    <script src="bestNearby/bundle.js"></script>
```

Comment:
Does the order of this matter for styling at all? It may be nice for the scroll of the page to match the order of the scripts.

Reply:
Good catch! I will switch the order, that was what I was going for but accidentally mixed them up at some point